spec/features/polls/results_spec.rb

```diff
@@ -1,6 +1,22 @@
 require 'rails_helper'
 
 feature 'Poll Results' do
+
+  scenario 'Does not show polls without enabled results' do
+    poll = create(:poll, results_enabled: false)
+    visit results_poll_path(poll)
+
+    expect(page).to have_content("Poll not available for public display")
+  end
+
+  scenario 'Does not show incomplete polls' do
+    poll = create(:poll, ends_at: Time.new.yesterday)
```

Comment:
Please @raul-fuentes always remember to use `Date.current` and `Time.current` as both use take into account the current timezone and avoid flaky specs that depend on the time of day when they're executed.
```
irb(main):001:0> Time.new.yesterday
=> 2018-03-18 15:28:16 +0100
irb(main):002:0> Time.current.yesterday
=> Sun, 18 Mar 2018 15:28:19 CET +01:00
```